src/web/components/Pagination/index.js

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import COLOR from '../../constant';
+import Button from '../Button';
+
+class Pagination extends React.Component {
+  constructor(props) {
+    super(props);
+    this.nextPageHandler = () => {
+      this.props.onChange(this.props.currentPage + 1);
```

Comment:
You can destructure in all the function your `props` to create something more readable 😊